src/Common/CosmosClient.ts

```diff
@@ -85,8 +85,7 @@ export function client(): Cosmos.CosmosClient {
     userAgentSuffix: "Azure Portal"
   };
 
-  // In development we proxy requests to the backend via webpack. This is removed in production bundles.
```

Comment:
I'm a bit concerned about this. What is the possible negative side effect of this if-statement not being removed in production bundles?